### PR TITLE
add support for hosts from inputs-inprogress

### PR DESCRIPTION
**Pull request by EarthmanT**

### CODE REVIEW

File: cloudify_ansible/tasks.py [hunk 1]
```diff
@@ -12,107 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from cloudify.decorators import operation
```
Comment (mabuaisha) [inline cloudify_ansible/tasks.py]: @EarthmanT Do we still need to use operation ?

File: examples/compute-blueprint.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      ip: { get_input: webserver_vm_ip }
+      agent_config:
+        install_method: none
+
+  dbserver_vm:
+    type: cloudify.nodes.Compute
+    properties:
+      ip: { get_input: dbserver_vm_ip }
+      agent_config:
+        install_method: none
```
Comment (mabuaisha) [inline examples/compute-blueprint.yaml]: @EarthmanT if both of the above compute node does not require agent installation and all what we need from them are just ips why not just depend on the ips input without even create those two nodes

File: cloudify_ansible/tasks.py (hunk 1 above)
Comment (EarthmanT) [reply cloudify_ansible/tasks.py]: @mabuaisha there have been discussions in R&D about making it mandatory again.

File: examples/compute-blueprint.yaml (hunk 2 above)
Comment (EarthmanT) [reply examples/compute-blueprint.yaml]: This is an example.

File: cloudify_ansible/tasks.py (hunk 1 above)
Comment (mabuaisha) [reply cloudify_ansible/tasks.py]: Ok

File: examples/compute-blueprint.yaml (hunk 2 above)
Comment (mabuaisha) [reply examples/compute-blueprint.yaml]: 👍

**Pull request merged**